utils/prepare-commit-msg.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+# This script is intended to be used a a git hook and will prepend the ticket
+# number to a commit message in the correct format for Github/Gitlab to parse.
+#
+# To use, create a shortcut to this file in .git/hooks called
+# 'prepare-commit-msg' e.g. from top folder of your project:
+#     cd .git/hooks
+#     ln -s ./utils/prepare-commit-msg.py prepare-commit-msg
+#
+# TODO - the above doesn't work on Windows
+
+import sys
+import re
+from subprocess import check_output
+
+# By default, the hook will check to see if the branch name starts with
+# 'issue-' and will then prepend whatever follows in the commit message.
```

Comment:
Just noting that this would be different to current practice? (Where the branch name is just the issue number) Not that this has ever been a strict requirement.

Not a problem, arguably having `issue-` is nicer/better.